fs/erofs/Config.in

```diff
@@ -6,10 +6,76 @@ config BR2_TARGET_ROOTFS_EROFS
 
 if BR2_TARGET_ROOTFS_EROFS
 
+choice
+	prompt "Compression algorithm"
+	default BR2_TARGET_ROOTFS_EROFS_LZ4HC
+	help
+	  Select the EROFS compression algorithm to use.
+
+config BR2_TARGET_ROOTFS_EROFS_NONE
+	bool "no compression"
+
+config BR2_TARGET_ROOTFS_EROFS_LZ4
+	bool "lz4"
+
 config BR2_TARGET_ROOTFS_EROFS_LZ4HC
-	bool "lz4hc compression"
+	bool "lz4hc"
+
+config BR2_TARGET_ROOTFS_EROFS_LZMA
+	bool "lzma"
+
+endchoice
+
+if BR2_TARGET_ROOTFS_EROFS_LZ4HC
+
+config BR2_TARGET_ROOTFS_EROFS_LZ4HC_LEVEL
+	int "lz4hc compression level"
+	default 9
+	range 1 12
+	help
+	  Specify the compression level for lz4hc compression.
+
+endif # BR2_TARGET_ROOTFS_EROFS_LZ4HC
+
+if BR2_TARGET_ROOTFS_EROFS_LZMA
+
+config BR2_TARGET_ROOTFS_EROFS_LZMA_LEVEL
+	int "lzma compression level"
+	default 6
+	range 0 109
+	help
+	  Specify the compression level for lzma compression.
+	  Values from 0 to 9 are used for the standard compression,
+	  values from 100 to 109 are used for the extreme compression.
+
+endif # BR2_TARGET_ROOTFS_EROFS_LZMA
+
+config BR2_TARGET_ROOTFS_EROFS_DEDUPE
```

Comment:
Can we make this dependent on `!BR2_TARGET_ROOTFS_EROFS_NONE`?

Then we'd not need the `ifneq ($(BR2_TARGET_ROOTFS_EROFS_NONE),y)` check below I guess.